values.yaml

```diff
@@ -29,7 +34,7 @@ api:
   password: storageos
 service:
   name: storageos
-  namespace: kube-system
+  namespace: default
```

Comment:
I think we should create a new namespace here itself, since we are moving out of kube-system.